include/cupla/math/Comparison.hpp

```diff
@@ -0,0 +1,45 @@
+/* Copyright 2020 Rene Widera
+ *
+ * This file is part of cupla.
+ *
+ * cupla is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * cupla is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with cupla.
+ * If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+
+#pragma once
+
+#include "cupla/math/Common.hpp"
+#include "cupla/types.hpp"
+
+namespace cupla
+{
+inline namespace CUPLA_ACCELERATOR_NAMESPACE
+{
+inline namespace device
+{
+inline namespace math
+{
+
+    //! Calculates the smaller value of two arguments.
+    CUPLA_BINARY_MATH_FN( min, alpaka::math::ConceptMathMin, Min )
+
+    //! Calculates the larger value of two arguments.
+    CUPLA_BINARY_MATH_FN( max, alpaka::math::ConceptMathMax, Max )
```

Comment:
Not a real issue, but I would interpret `Comparison.hpp` as things like `>, <, ==` and not `min` and `max`.

Reply:
I used the naming from PMacc.